### PR TITLE
Archive

### PR DESCRIPTION
**Pull request by Guntrisoft**

### CODE REVIEW

File: app/views/articles/archives.html.erb [hunk 1]
```diff
@@ -1,24 +1,31 @@
...
-  <div class="post">
-  <p><%= t(".no_articles_found")%></p>
+<div class="l-results">
+  <div class="l-constrained">
+    <h1 class="l-results__title">Archive</h1>
+
+    <%- if @articles.empty? %>
+      <p><%= t(".no_articles_found")%></p>
+    <%- else %>
+      <%- for article in @articles %>
```
Comment (tomas-stefano) [inline app/views/articles/archives.html.erb]: A doubt, why not use `@articles.each`? :)

**Comment by Guntrisoft** [comment]
@benbarnett @baphled

File: app/views/articles/archives.html.erb (hunk 1 above)
Comment (baphled) [inline app/views/articles/archives.html.erb]: Agreed

File: app/assets/stylesheets/layouts/_index.scss [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  margin-top: -5px;
+  @include font-size(30);
+  text-align: center;
+  width: 100%;
+
+  &:link,
+  &:hover,
+  &:focus,
+  &:visited {
+    color: $white;
```
Comment (benbarnett) [inline app/assets/stylesheets/layouts/_index.scss]: Shouldn't we have a different visual style for focus/active?

File: app/assets/stylesheets/layouts/_tiles.scss [hunk 3]
```diff
@@ -58,3 +57,16 @@
     }
   }
 }
+
+.l-tile__archive {
+  float: right !important;
```
Comment (benbarnett) [inline app/assets/stylesheets/layouts/_tiles.scss]: `!important` needed?

File: app/assets/stylesheets/layouts/_tiles.scss [hunk 4]
```diff
@@ -58,3 +57,16 @@
...
+
+  @include respond-to($mq-m) {
+    .flexbox & {
+      margin-left: auto;
+      float: none;
+    }
+
+    padding-left: 5px !important;
```
Comment (benbarnett) [inline app/assets/stylesheets/layouts/_tiles.scss]: `!important` needed?

File: app/assets/stylesheets/layouts/_index.scss (hunk 2 above)
Comment (Guntrisoft) [inline app/assets/stylesheets/layouts/_index.scss]: There is an underline on focus/hover

File: app/assets/stylesheets/layouts/_tiles.scss (hunk 3 above)
Comment (Guntrisoft) [inline app/assets/stylesheets/layouts/_tiles.scss]: I wrangled with it for a while, there's a fair bit of nth-child stuff to do with the .l-tile which is overriding it - this was the easiest and quickest way out! - I'll take another look it tomorrow morning.

**Comment by Guntrisoft** [comment]
@benbarnett let me know if this is now good to go

File: app/views/articles/archives.html.erb [hunk 5]
```diff
@@ -1,24 +1,31 @@
...
+        </div>
+      <%- end %>
+    <%- end %>
+  </div>
+
+  <% if @articles.total_pages > 1 %>
+    <nav class="l-results-pagination">
+      <p class="l-results-pagination__text">Page <%= @articles.current_page %> of <%= @articles.total_pages %></p>
+
+      <% if !@articles.first_page? %>
```
Comment (benbarnett) [inline app/views/articles/archives.html.erb]: Probably cleaner to use `unless @articles.first_page?` here

**Comment by Guntrisoft** [comment]
@benbarnett Ben has verbally said it's good to go...

**Pull request merged**